api/v1/env.go

```diff
@@ -302,6 +301,14 @@ func (s *Store) GetEnv() []corev1.EnvVar {
 			Name:  "SQL_SET_DEFAULT_SESSION_VARIABLES",
 			Value: "0",
 		},
+		{
+			Name:  "INSTALL_LOCALE",
```

Comment:
Does this will clash if it's define an extra env?

Reply:
They will get defined twice, need to check what kubernetes is then doing. But we should not allow overwriting variables defined from the operator. Maybe log a warning that a variable is declared but operator overwrites it.

Reply:
```You can technically define the same environment variable multiple times, and it will take the last one on the list.```

Reply:
ok got it thanks.
Maybe lot a new issue to tackle this situation later.